### ISSUE TITLE
Problem working with RT5

### ISSUE THREAD

**Issue by fapgomes** [bheisig/rt-extension-referenceidoitobjects#7]
The plugin doesn't work with RT5.

**Comment by bheisig** [comment]
Yes, you're right. At the moment, there are no plans to add new or improve existing features. It's only maintained with bug and security fixes. Upgrading it to v5 results in a huge effort. But don't hesitate to create a pull request if you like.
